crates/apub/assets/lemmy/activities/community/block_user.json

```diff
@@ -9,5 +9,6 @@
   ],
   "target": "http://enterprise.lemmy.ml/c/main",
   "type": "Block",
+  "expires": "2021-11-01T12:23:50.151874+00:00",
```

Comment:
Not sure I'm doing this correctly. Its an optional field so maybe I didn't need to add it at all.

Reply:
Better to add it, to test that parsing works correctly.